PacketDotNet/DrdaCodepointType.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
```

Comment:
Please include the licence header here and your information for the copyright statement.